docs/02-developers/05-smart-contracts/05-foundry/configure-foundry-rootstock.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: Configure Foundry for Rootstock
+sidebar_position: 102
+title: Configure Foundry for Rootstock
+description: "Learn how to configure your Foundry project for development on Rootstock testnet and mainnet"
+tags: [guides, developers, smart contracts, rsk, rootstock, foundry, dApps, ethers]
+---
+
+### Environment Configuration
+
+Once you have an account with a private key, create a `.env` file in the root of the foundry project and add the variables. 
+
+Foundry automatically loads a `.env` file present in the project directory.
+
+The `.env` file should follow this format:
+
```

Comment:
Add a note on how to get the RPC API URL in /developers/rpc-api/setup/

Reply:
okay noted, I will add this one asap